tests/posix/c_lib_ext/src/getentropy.c

```diff
@@ -5,11 +5,12 @@
  */
 
 #include <unistd.h>
+
 #include <zephyr/ztest.h>
 
 ZTEST(posix_c_lib_ext, test_getentropy_too_large)
 {
-	uint8_t buf[256 + 1] = {0};
+	uint8_t buf[GETENTROPY_MAX + 1] = {0};
```

Comment:
This seems unrelated to this commit?

Reply:
Also a drive-by - I noticed that this had a standard limit so added it. I can remove that and we can continue using hard-coded values.

Reply:
Fixed (in staging)